generators/entity/index.js

```diff
@@ -738,7 +738,7 @@ class EntityGenerator extends BaseBlueprintGenerator {
                         'byte[]',
                         'ByteBuffer'
                     ].includes(fieldType);
-                    if (['sql', 'mongodb', 'couchbase', 'no'].includes(context.databaseType) && !nonEnumType) {
+                    if (['sql', 'mongodb', 'couchbase', 'no', 'cassandra'].includes(context.databaseType) && !nonEnumType) {
```

Comment:
Since all databases types are not included, the `if` is not necessary anymore: 
```
field.fieldIsEnum = [
                        'String',
                        'Integer',
                        'Long',
                        'Float',
                        'Double',
                        'BigDecimal',
                        'LocalDate',
                        'Instant',
                        'ZonedDateTime',
                        'Duration',
                        'UUID',
                        'Boolean',
                        'byte[]',
                        'ByteBuffer'
                    ].includes(fieldType);
```